src/core/starboard/handle.rs

```diff
@@ -180,9 +176,9 @@ impl<'this, 'bot> RefreshStarboard<'this, 'bot> {
                 if matches!(get_status(&why), Some(404)) {
                     StarboardMessage::delete(&self.refresh.bot.pool, sb_msg.starboard_message_id)
                         .await?;
-                    return Ok(retry_on_err);
+                    Ok(retry_on_err)
                 } else {
-                    return Err(why.into());
+                    Err(why.into())
```

Comment:
This looks wrong. `return` will return out of the function, this will just return the value from the block.

Reply:
clippy said it was fine

Reply:
Yeah, it might not cause an error, but it changes behavior.

Reply:
wait nevermind, it doesn't change behavior.